### PR TITLE
Stash scan email in sessionStorage, auto-populate subscribe modal email input with it

### LINKED ISSUE #395: Automatically populate email address in subscribe modal
If the user clicks subscribe after a scan, it would be nice to auto-fill the email address they just scanned. We can stash the email in sessionStorage at scan-time and use that to populate the textbox when the modal is shown.

### PR DESCRIPTION
**Pull request by nhnt11**
Fixes #395

### CODE REVIEW

**Comment by groovecoder** [comment]
LGTM.

File: public/js/monitor.js [hunk 1]
```diff
@@ -145,6 +145,10 @@ function openModalWindow() {
   ga_sendPing("SignUp");
   const subscribeModal = document.getElementById("subscribe-modal");
   document.body.classList.add("show-subscribe-modal");
+  const currentEmail = sessionStorage.getItem("current_email");
```
Comment (pdehaan) [inline public/js/monitor.js]: Does sessionstorage throw any error if it, or cookies are disabled?

Comment (groovecoder) [reply public/js/monitor.js]: No errors thrown without sessionStorage. I'm going to merge for a release today.

**Comment by groovecoder** [comment]
Dang, this didn't automatically close #395. I think you have to put "fix #395" in the commit message somewhere? Not just the PR comments?

**Comment by groovecoder** [comment]
Whoa, this PR was to merge this commit into the `385` branch - not `master`.

**Pull request merged**